org-ql.el

```diff
@@ -60,6 +60,11 @@
 Like `org-clock-line-re', but matches the timestamp range in a
 match group.")
 
+(defconst org-ql-log-done-regexp
+  (rx bol "- State \"DONE\"" (1+ blank) "from \"WAITING\"" (1+ blank) (group (1+ not-newline)))
```

Comment:
I think logbooks always use the `-` character as their bullet character, but we should make sure of that before merging.